components/text/styles.js

```diff
@@ -1,5 +1,8 @@
 import styled from 'styled-components'
+import COLORS from 'shared/colors'
 
-const View = styled.div``
+const View = styled.div`
+  color: ${props => props.color ? props.color : COLORS.dark};
```

Comment:
instead of `shared/colors` we should see how to use `styled-system` and `styled-components` themes